content/docs/command-reference/add.md

```diff
@@ -59,11 +59,9 @@ Summarizing, the result is that the target data is replaced by small
 [`.dvc` files](/doc/user-guide/dvc-files-and-directories#dvc-files) that can be
 easily tracked with Git.
 
-> Note that `.dvc` files can be considered _orphan stages_, because they have no
-> <abbr>dependencies</abbr>, only outputs. These are treated as _always changed_
-> by `dvc status` and `dvc repro`, which always executes them. See
-> [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file) to learn
-> more about stages.
+> Note that `.dvc` files can be considered don't have <abbr>dependencies</abbr>,
+> only outputs. These are treated as _always changed_ by `dvc status` and
+> `dvc repro`, which always executes them.
```

Comment:
Looks unrelated to granularity?

Reply:
Yeah I had to re-read parts of the refs to figure out how to express the granularity which led me to discover misc. problems and it's just faster to throw them into the PR than add them to a separate one and remember that it's being fixed in parallel, etc.

Reply:
Makes it harder to review though :slightly_smiling_face: But do as you see fit.

Reply:
Looks like a wrong sentence too! Fixed in b01ccd95